setup.py

```diff
@@ -1,21 +1,32 @@
-from setuptools import setup
+import setuptools
 import os
 
-VERSION = '0.10'
+VERSION = '0.11'
 
-setup(
+install_requires = []
+py2_extras = ["argparse", "simplejson"]
+extras_require = {":python_version<'3'": py2_extras}
+
+if int(setuptools.__version__.split('.')[0]) < 18:
```

Comment:
I think we can drop the check as setuptools is very old, debian old stable has 33.